packages/backend/src/auto-peering/routes.ts

```diff
@@ -1,13 +1,27 @@
 import { AppContext } from '../app'
 import { BaseService } from '../shared/baseService'
+import { errorToCode, errorToMessage, isAutoPeeringError } from './errors'
 import { AutoPeeringService } from './service'
 
+interface PeeringRequestArgs {
+  staticIlpAddress: string
+  ilpConnectorAddress: string
+  asset: { code: string; scale: number }
```

Comment:
Would it be easier if we ask for an array of assets?

Reply:
replied: https://github.com/interledger/rafiki/pull/1849#discussion_r1324279743